packages/editor-ui/src/api/settings.ts

```diff
@@ -1,25 +1,20 @@
-import { IDataObject } from 'n8n-workflow';
-import { IRestApiContext, IN8nPrompts, IN8nValueSurveyData, IN8nUISettings, IPersonalizationSurveyAnswers } from '../Interface';
+import { IRestApiContext, IN8nPrompts, IN8nValueSurveyData, IN8nUISettings } from '../Interface';
 import { makeRestApiRequest, get, post } from './helpers';
 import { TEMPLATES_BASE_URL } from '@/constants';
 
 export async function getSettings(context: IRestApiContext): Promise<IN8nUISettings> {
 	return await makeRestApiRequest(context, 'GET', '/settings');
 }
 
-export async function submitPersonalizationSurvey(context: IRestApiContext, params: IPersonalizationSurveyAnswers): Promise<void> {
-	await makeRestApiRequest(context, 'POST', '/user-survey', params as unknown as IDataObject);
+export async function getPromptsData(instanceId: string, userId: string): Promise<IN8nPrompts> {
+	return await get(TEMPLATES_BASE_URL, '/prompts', {}, {'n8n-instance-id': instanceId, 'n8n-user-id': userId});
```

Comment:
why do the functions in here use `async` and then return `await`?

can't we omit the `async` and simply return the promis from the `get | post`?

Reply:
ya you are right